AUTHORS

```diff
@@ -8,3 +8,4 @@ Felix Frank
 James Shubin
 Julien Pivotto
 Paul Morgan
+Corey Osman
```

Comment:
We'd love to have you in the core AUTHORS file, but we need to wait until you've got some more core patches under your belt first. Looking forward to seeing you there, and let us know if you need more help getting started.

We've got lots of tagged "new contributor" patches: https://github.com/purpleidea/mgmt/issues?q=is%3Aissue+is%3Aopen+label%3Amgmtlove

Reply:
I was following the instructions in the file:

```
This is a list of authors/contributors to the mgmt project.
If you're a contributor, please send a patch with your name.
If you appreciate the work of one of the contributors, thank them a beverage!
For a more exhaustive list please run: git log --format='%aN' | sort -u
This list is sorted alphabetically by first name.
```

Am I a contributor?  Yes as I submitted code
Does this file say anything about core contributor?  No

Reply:
@logicminds You're right. Sorry, it's a sort of unwritten rule. We should reword it.

You *are* contributing. Thank you for that!

Reply:
removed name